source/reference.txt

```diff
@@ -1,9 +1,53 @@
-.. place holder for the reference section TOC
-
 =========
 Reference
 =========
 
+:ref:`c2c-mongosync`
+   The primary process used in {+c2c-product-name+}.
+
+:ref:`c2c-config`
+   Configure :program:`mongosync` instances at startup using a
+   configuration file.
+
+:ref:`c2c-api`
+   The API endpoints to interact with ``mongosync``. 
```

Comment:
[suggestion]

Consider dropping "The" here.

Reply:
Done!